beets/importer.py

```diff
@@ -547,9 +547,22 @@ def _emit_imported(self, lib):
         plugins.send('album_imported', lib=lib, album=self.album)
 
     def emit_created(self, session):
-        """Send the `import_task_created` event for this task.
+        """Send the `import_task_created` event for this task and return a list
+        of tasks which might be returned by plugins.
```

Comment:
Maybe the method name needs to change too, since this now does more than just emit the event. Maybe `get_created`? That's a little opaque too… maybe there's a better idea out there.